x-pack/test/screenshot_creation/apps/ml_docs/data_frame_analytics/outlier_detection.ts

```diff
@@ -0,0 +1,132 @@
+
+      await ml.testExecution.logTestStep('take screenshot');
+      await mlScreenshots.takeScreenshot('logs-transform-preview', screenshotDirectories);
+    });
+
+    it('wizard screenshots', async () => {
+      await ml.testExecution.logTestStep('navigate to data frame analytics list');
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToDataFrameAnalytics();
+
+      await ml.testExecution.logTestStep('open outlier detection job in wizard');
+      await ml.dataFrameAnalyticsTable.waitForAnalyticsToLoad();
+      await ml.dataFrameAnalyticsTable.filterWithSearchString(outlierJobConfig.id as string, 1);
+      await ml.dataFrameAnalyticsTable.cloneJob(outlierJobConfig.id as string);
+      await ml.dataFrameAnalyticsCreation.assertSourceDataPreviewExists();
+      await ml.dataFrameAnalyticsCreation.assertIncludeFieldsSelectionExists();
+
+      await ml.testExecution.logTestStep('take screenshot');
+      await mlScreenshots.takeScreenshot('weblog-outlier-job-1', screenshotDirectories);
+
+      await ml.testExecution.logTestStep('scroll to scatterplot matrix and take screenshot');
+      await ml.dataFrameAnalyticsCreation.assertScatterplotMatrixLoaded();
+      await ml.dataFrameAnalyticsCreation.scrollScatterplotMatrixIntoView();
+      await mlScreenshots.takeScreenshot('weblog-outlier-scatterplot', screenshotDirectories);
+    });
+
+    it('results view screenshots', async () => {
+      await ml.testExecution.logTestStep('navigate to data frame analytics list');
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToDataFrameAnalytics();
+
+      await ml.testExecution.logTestStep('open job results view');
+      await ml.dataFrameAnalyticsTable.refreshAnalyticsTable();
+      await ml.dataFrameAnalyticsTable.filterWithSearchString(outlierJobConfig.id as string, 1);
+      await ml.dataFrameAnalyticsTable.openResultsView(outlierJobConfig.id as string);
+      await ml.dataFrameAnalyticsResults.assertOutlierTablePanelExists();
+      await ml.dataFrameAnalyticsResults.assertResultsTableExists();
+      await ml.dataFrameAnalyticsResults.assertResultsTableNotEmpty();
+
+      await ml.testExecution.logTestStep('fold scatterplot section and take screenshot');
+      await ml.dataFrameAnalyticsResults.expandScatterplotMatrixSection(false);
+      await mlScreenshots.removeFocusFromElement();
+      await mlScreenshots.takeScreenshot('outliers', screenshotDirectories);
+
+      await ml.testExecution.logTestStep('scroll to scatterplot matrix and take screenshot');
+      await ml.dataFrameAnalyticsResults.expandScatterplotMatrixSection(true);
+      await mlScreenshots.removeFocusFromElement();
+      await ml.dataFrameAnalyticsResults.assertScatterplotMatrixLoaded();
+      await ml.dataFrameAnalyticsResults.scrollScatterplotMatrixIntoView();
+      await mlScreenshots.takeScreenshot('outliers-scatterplot', screenshotDirectories);
```

Comment:
Nit - could ad the data set name in here for consistency with the others e.g. `weblog-outliers-scatterplot` (although that would clash for a screenshot used from the outlier wizard).

Reply:
See my comment above: we can discuss screenshot file names with docs folks in a follow-up.